LDAP-Auth/Config/configPage.html

```diff
@@ -8,108 +8,74 @@
 
         <div data-role="content">
             <div class="content-primary">
-                <p><i>Note:</i> Making changes to this configurationr requires a restart of Jellyfin.</p>
                 <form class="esqConfigurationForm">
-                    <ul class="ulForm" data-role="listview">
-                        <li>
-                            <label for="txtLdapServer">
-                                LDAP Server:
-                            </label>
-                            <input type="text" id="txtLdapServer" name="txtLdapServer" />
-                            <div class="fieldDescription">
-                                The Servername for the LDAP server you wish to use for Authentications
+                    <div class="verticalSection verticalSection-extrabottompadding">
+                        <div class="sectionTitleContainer flex align-items-center">
+                            <h2 class="sectionTitle">LDAP Settings:</h2>
+                            <a is="emby-linkbutton" class="raised button-alt headerHelpButton" target="_blank" href="https://github.com/jellyfin/jellyfin-plugin-ldapauth">${Help}</a>

```

Comment:
`emby-linkbutton` is going away. Use `emby-button` instead

Reply:
ok, fixed it.